### PR TITLE
Allow initialization of char ptr array

### PR DESCRIPTION
**Pull request by kant2002**

### CODE REVIEW

File: Cesium.CodeGen.Tests/CodeGenArrayTests.cs [hunk 1]
```diff
@@ -197,4 +197,21 @@ int main(void)
...
+    public Task ConstByteArrayTest() => DoTest(@"
+int main() {
+    const char a[1] = { 255 };
+}");
+
+    [Fact]
+    public Task ArrayCharComparison() => DoTest(@"
+int main() {
+    const char a[1] = { 'A' };
+    if (a[0] != 'D') {
```
Comment (slyshykO) [inline Cesium.CodeGen.Tests/CodeGenArrayTests.cs:211]: Why is there a check for 'D' despite the array being initialized with 'A'?

Comment (kant2002) [reply Cesium.CodeGen.Tests/CodeGenArrayTests.cs:211]: I test what’s generated. I need comparison with any char literal. Does not matter what.

Comment (ForNeVeR) [reply Cesium.CodeGen.Tests/CodeGenArrayTests.cs:211]: Yeah. This doesn't matter. We don't even run this code. It's for the purpose of dumping the codegen results. So, more variety (within some measure of course) — more chances the tests will catch unexpected codegen changes. This is good.

**Review by ForNeVeR — APPROVED** [review]
Thanks!

**Pull request merged**